readme.md

```diff
@@ -201,6 +201,25 @@ got('todomvc.com', {
 }, function () {});
 ```
 
+### Unix Domain Sockets
+
+Requests can also be sent via unix domain sockets. Use the following URL scheme: `PROTOCOL://unix:SOCKET:PATH`.
```

Comment:
Can you linkify `unix domain sockets` to an article/page where the user can read more about unix domain sockets?